estmator_project/estmator_project/templates/menu.html

```diff
@@ -0,0 +1,168 @@
+                    <button type="submit" class="btn btn-default">Submit</button>
+                </form>
+            </div>
+            <div class="modal-footer"></div>
+        </div>
+    </div>
+</div>
+
+<div class="modal fade" id="edit_quote" tabindex="-1" role="dialog">
+    <div class="modal-dialog" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+                <h4 class="modal-title">Edit Quote</h4>
+            </div>
+            <div class="modal-body text-center">
+                <form id="edit_quote_form" action="{% url 'quote' %}" method="post">{% csrf_token %}
+                    <div id="edit_quote_form_list"></div>
+                    <div id="edit_quote_form_contents"></div>
+                    <!-- <div id="edit_quote_form_contents">{{ client_form }}</div> -->
+                    <button type="submit" class="btn btn-default">Submit</button>
+                </form>
+            </div>
+            <div class="modal-footer"></div>
+        </div>
+    </div>
+</div>
+
+<div class="modal fade" id="new_client" tabindex="-1" role="dialog">
+    <div class="modal-dialog" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+                <h4 class="modal-title">New Client</h4>
+            </div>
+            <div class="modal-body text-center">
+                <form id="new_client_form" action="{% url 'api_client' %}" method="post">{% csrf_token %}
+                    {{ client_form }}
+                    <button type="submit" class="btn btn-default">Submit</button>
+                </form>
+            </div>
+            <div class="modal-footer"></div>
+        </div>
+    </div>
+</div>
+
+<div class="modal fade" id="edit_client" tabindex="-1" role="dialog">
+    <div class="modal-dialog" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+                <h4 class="modal-title">Edit Client</h4>
+            </div>
+            <div class="modal-body text-center">
+                <form id="edit_client_form" action="{% url 'api_client' %}" method="post">{% csrf_token %}
+                    <div id="edit_client_form_list"></div>
+                    <div id="edit_client_form_contents">{{ client_form }}</div>
+                    <button type="submit" class="btn btn-default">Submit</button>
+                </form>
+            </div>
+            <div class="modal-footer"></div>
+        </div>
+    </div>
+</div>
+
+{% endblock %}
```

Comment:
The amount of inline js in this file makes me wonder if you might not want to start leaning toward building a js library to allow you to lift it all out of your templates.